universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx

```diff
@@ -16,7 +16,7 @@ export const EmojiPanelWithFakes = ({securityCodeWithFakes, onEmojiClicked}: Emo
   return (
     <div>
       <p>Security code</p>
-      <ul style={{display: 'flex', listStyle: 'none'}}>
+      <ul className={'universal-login-emoji-default'}>
```

Comment:
That is going to be a bit more complex:
- we should use pattern started with WalletSelector
- we could rename `getWalletSelectorClass(className)` -> `getClassNameFor(customClassName)`
- we can add class to `<div>` above, ie: `<div className='universal-login-emoji'/>
- now we select this components in css like this:
  - Generic styles: `.universal-login-emoji ...`
  - Default styles:  `.universal-login-emoji .universal-login-default ...`
  - Custom styles: `.universal-login-emoji .some-custom-class ...`